src/styles/api-flows.css

```
@@ -1527,3 +1668,453 @@
+}
+
+.af-prompt-section--collapsed .af-prompt-section-body {
+    display: none;
+}
+
+.af-prompt-section-view {
+    font-size: 0.8125rem;
+    line-height: 1.65;
+    color: var(--main-text-color);
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)